src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMapping.java

```diff
@@ -49,6 +55,25 @@ public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData) {
         }
     }
 
+    public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData, ReferenceDataServiceHolder referenceDataServiceHolder) {
+        log.info("Get the next hearing channel {}", referenceDataServiceHolder.isAdjournmentFlagEnabled());
```

Comment:
The difficulty I have with understanding this code is "hearingChannel" and "nextHearingChannel" - the latter implies adjournment? It isn't very clear and requires some domain knowledge. If some naming could be improved to indicate more clearly when we are dealing with adjournment, I think that would help with understanding this bit of code. It could be as simple as inserting ForAdjournment into method names for example.

Reply:
Renamed the method name **getAdjournmentNextHearingChannel**